populus/contracts/provider.py

```diff
@@ -83,26 +83,10 @@ def is_contract_available(self, contract_identifier):
         return True
 
     def are_contract_dependencies_available(self, contract_identifier):
-        full_dependency_graph = get_shallow_dependency_graph(
-            self.chain.project.compiled_contract_data,
-        )
-        contract_dependencies = get_recursive_contract_dependencies(
-            contract_identifier,
-            full_dependency_graph,
-        )
+        cdata = self.get_contract_data(contract_identifier)
+        contract_dependencies = cdata['ordered_dependencies']
```

Comment:
I made this switch to cached dependencies early in the implementation because I thought it was necessary. Now, I'm wondering if I should revert all of that.